ops/ccs-ops-misc/load_test/v1/baseline_tests.py

```diff
@@ -0,0 +1,131 @@
+import random
+from typing import Dict
+import urllib3
+import common.config as config
+import common.data as data
+import common.errors as errors
+import common.test_setup as setup
+from common.url_path import create_url_path
+import common.validation as validation
+from locust import HttpUser, task, events, tag
+
+client_cert = setup.getClientCert()
+server_public_key = setup.loadServerPublicKey()
+setup.disable_no_cert_warnings(server_public_key, urllib3)
+setup.set_locust_env(config.load())
+
+mbis = data.load_mbis()
+eob_ids = data.load_bene_ids()
+last_updated = data.get_last_updated()
+cursor_list = data.load_cursors("v1")
+
+class BFDUser(HttpUser):
```

Comment:
Could you add some documentation for this class please?

Reply:
Yeah, I'll add some documentation to both regression suites and to their class-level methods.

Reply:
@lsmitchell I've added documentation comments to both files; let me know if you were thinking of adding external documentation or if you think that the file comments I've added are sufficient.

Reply:
That should be fine, just wanted some small explanation of what the tests are doing in the file.